nomad/state/state_store.go

```diff
@@ -374,14 +374,14 @@ func (s *StateStore) UpsertPlanResults(msgType structs.MessageType, index uint64
 
 	// Mark nodes as ineligible.
 	for _, nodeID := range results.IneligibleNodes {
-		s.logger.Warn("marking node as ineligible due to multiple plan rejections, refer to https://www.nomadproject.io/s/port-plan-failure for more information", "node_id", nodeID)
+		s.logger.Warn("Marking node as ineligible due to multiple plan rejections, refer to https://www.nomadproject.io/s/port-plan-failure for more information", "node_id", nodeID)
 
 		nodeEvent := structs.NewNodeEvent().
 			SetSubsystem(structs.NodeEventSubsystemScheduler).
 			SetMessage(NodeEligibilityEventPlanRejectThreshold)
 
 		err := s.updateNodeEligibilityImpl(index, nodeID,
-			structs.NodeSchedulingIneligible, results.UpdatedAt, nodeEvent, txn)
+			structs.NodeSchedulingIneligible, "Marking node as ineligible due to multiple plan rejections", results.UpdatedAt, nodeEvent, txn)
```

Comment:
I'd tighten up the description if you're outputting it in the table output 
```suggestion
			structs.NodeSchedulingIneligible, "ineligible due to multiple plan rejections", results.UpdatedAt, nodeEvent, txn)
```